fstat/controller.py

```diff
@@ -76,7 +77,17 @@ def overall_summary():
         filters.append(FailureInstance.branch == branch)
 
     failure_instances = FailureInstance.query.filter(*filters)
+
     failures = Counter([x.failure for x in failure_instances])
+    if request.endpoint == 'api:failures':
+        failures_ = []
+        for failure, count in failures.most_common():
+            failures_.append({
+                "signature": "http://git.gluster.org/cgit/glusterfs.git/tree/" + failure.signature,
```

Comment:
Don't do this. I provide this on the web UI as convenience method. We should not be assuming that signature = failure where test failed.